Detectors/ITSMFT/ITS/tracking/CMakeLists.txt

```diff
@@ -36,6 +37,15 @@ o2_add_library(ITStracking
                        O2::ITSMFTReconstruction
                        O2::DataFormatsITS)
 
+
+o2_add_library(ITSTrackingInterface
+               TARGETVARNAME targetName
+               SOURCES src/TrackingInterface.cxx
+               PUBLIC_LINK_LIBRARIES
```

Comment:
Without really compiling and trying it, the PR looks good, except for one thing that I do not understand:
For the GPUTrackingInterfaces, the idea is to link all the internal libraries as private, such that not all GPU headers are exposed to everything that links to the GPUTrackingInterface.
But here you link as PUBLIC. Is this just a typo? Otherwise I do not understand why the ITSTrackingInterface is a separate library and not simply part of ITSTracking.

Reply:
Indeed these can be privately linked.
In general, the ITSTrackingInterface is not part of ITStracking so to avoid a circular dependency in ITSRecoWorkflow as both link vs GPU libs.